src/ontology/cl-edit.owl

```diff
@@ -17603,13 +17606,15 @@ AnnotationAssertion(owl:deprecated obo:CL_0002371 "true"^^xsd:boolean)
 
 # Class: obo:CL_0002372 (myotube)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:dos") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "ISBN:0323052908") Annotation(oboInOwl:hasDbXref "PMID:22274696") obo:IAO_0000115 obo:CL_0002372 "A transversely striated, synctial muscle cell, formed by the fusion of myoblasts.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:dos") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "ISBN:0323052908") Annotation(oboInOwl:hasDbXref "PMID:22274696") obo:IAO_0000115 obo:CL_0002372 "A transversely striated, multinucleated syncytial muscle cell, formed by the fusion of myoblasts during muscle development.")
 AnnotationAssertion(terms:contributor obo:CL_0002372 <https://orcid.org/0000-0003-1980-3228>)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_0002372 "2010-09-24T01:13:01Z"^^xsd:dateTime)
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0002372 "myotubule")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0002372 "single cell sarcomere")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://www.sciencedirect.com/topics/medicine-and-dentistry/myotube") oboInOwl:hasExactSynonym obo:CL_0002372 "myotubule")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "FBbt:00005812") oboInOwl:hasExactSynonym obo:CL_0002372 "myofiber")
+AnnotationAssertion(Annotation(rdfs:comment "FBbt:00005812") oboInOwl:hasExactSynonym obo:CL_0002372 "myofibril")
```

Comment:
the comment needs to be a dbxref. The QI shouldn't have allowed it, what is worrying.